src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationWriter.cs

```diff
@@ -1853,7 +1854,7 @@ private string GenerateMembersElement(XmlMembersMapping xmlMembersMapping)
                         }
 
                         Writer.Write("if (pLength > ");
-                        Writer.Write(i.ToString(CultureInfo.InvariantCulture));
```

Comment:
Is this "safe", integer string conversion was rather culture specific, or at least I thought it was; and to the best of my knowledge everything in NetFx takes current culture in account unless specifically told not yo.